en/release-notes/upgrade-guide-v3.0.md

```diff
@@ -11,37 +11,31 @@ In order to ensure the smooth transition of an existing __Cocos Creator 2.4__ pr
 
 This __Cocos Creator 3.0 Preview__ version is close to the official version in terms of functionality, and can be used for new project establishment and feature pre-research. The next official version will be released around Feb 11 - Feb 17, 2021, and will further improve the performance and fix the problems at that time to ensure that everyone's new projects can proceed smoothly.
 
-This __Cocos Creator 3.0 Preview__ version uses a new future-oriented engine architecture, which will bring high-performance, data-oriented and load-balanced renderers to the engine, and seamlessly support Vulkan & Metal multi-backend rendering. In the future, it will also support mobile VR/AR and some Host platform. For a detailed introduction to the __Cocos Creator 3.0 Preview__, please go to [Official Website Update Instructions](https://cocos.com/creator).
+The __Cocos Creator 3.0__ uses a new future-oriented engine architecture, which will bring high-performance, data-oriented and load-balanced renderers to the engine, and seamlessly support Vulkan & Metal multi-backend rendering. In the future, it will also support mobile VR/AR and some Host platform. For a detailed introduction to the __Cocos Creator 3.0 Preview__, please go to [Official Website Update Instructions](https://cocos.com/creator).
 
 ## How to migrate (not supported in Preview version)
 
-Although **we do not recommend projects under development, especially projects that are about to go live, upgrade to 3.0**, we will still launch the 2.x resource import tool in the 3.0 official version. This tool will support the perfect import of old project resources and the auxiliary import of codes. Code-assisted import will convert **Javascript** into **TypeScript**, add component type declarations, attribute declarations and function declarations. The references of components in the scene will be preserved, and the code inside the function will be imported in the form of comments, which can reduce the developer's upgrade difficulty.
+Although **we do not recommend projects under development, especially projects that are about to go live, upgrade to 3.0**, we will still launch the 2.x resource import tool in the 3.0 official version. This tool will support the perfect import of old project resources and the auxiliary import of codes. Code-assisted import will convert **JavaScript** into **TypeScript**, and automatically add component type declarations, attribute declarations and function declarations. The references of components in the scene will be preserved, and the code inside the function will be imported in the form of comments, which can reduce the developer's upgrade difficulty.
```

Comment:
upgrade to 3.0**
-->
to upgrade to 3.0**

Reply:
we will still launch the 2.x resource import tool in the 3.0 official version.
-->
there will be a  2.x resource import tool when 3.0 is officially released.

Reply:
This tool will support the perfect import of old project resources and the auxiliary import of codes.
-->
This tool will support importing old projects, project resources, and project code.

Reply:
which can reduce the developer's upgrade difficulty.
-->
which can reduce the difficulty of upgrading.